### PR TITLE
Separate KrakenUniq test run

### PR DESCRIPTION
**Pull request by jfy133**
As when database unpacked it results in 17GB which is more than 14GB, (but sometimes works withGHA? ) separating allows us to ignore it if necessary

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -101,3 +101,36 @@ jobs:
         with:
           command: nextflow run ${GITHUB_WORKSPACE} -profile test_motus,docker --outdir ./results --databases ./database_motus.csv
           attempt_limit: 3
+
+  krakenuniq:
+    name: Test mOTUs with workflow parameters
```
Comment (Midnighter) [inline .github/workflows/ci.yml:106]:
```suggestion
    name: Test KrakenUniq with workflow parameters
```

File: conf/test_krakenuniq.config [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+----------------------------------------------------------------------------------------
+*/
+
+//
+// Separate test as KrakenUniq database can sometimes be too big for GHA
+//
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test to check mOTUs function'
```
Comment (Midnighter) [inline conf/test_krakenuniq.config:19]:
```suggestion
    config_profile_description = 'Minimal test to check KrakenUniq function'
```

**Review by Midnighter — APPROVED** [review]
Other than the profiler copy & paste errors this looks good.

**Comment by jfy133** [comment]
Close enough another flaky download

**Pull request merged**